src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferDetail.cshtml.cs

```diff
@@ -40,27 +40,40 @@ public async Task<IActionResult> OnGetAsync(string id, string serviceid)
         ReturnUrl = Request.Headers["Referer"].ToString();
         LocalOffer = await _organisationClientService.GetLocalOfferById(serviceid);
         Name = LocalOffer.Name;
-        if (LocalOffer.Locations.Count != 0) ExtractAddressParts(LocalOffer.Locations.First());
+        if (LocalOffer.Locations != null && LocalOffer.Locations.Count != 0) ExtractAddressParts(LocalOffer.Locations.First());

```

Comment:
Consider using `Any()` rather than `Count != 0` (I think it makes the intention a little clearer).